core/modules/installer/installer.browser.pages.inc

```diff
@@ -0,0 +1,998 @@
+    'usage' => array('method' => 'usage', 'name' => t('Most installed'), 'default_sort' => 'desc'),
+    'title' => array('method' => 'title', 'name' => t('Title'), 'default_sort' => 'asc'),
+    'latest_release' => array('method' => 'latest_release', 'name' => t('Latest release'), 'default_sort' => 'desc'),
+  );
+
+  if ($full) {
+    $sort_options['type'] = array('method' => 'type', 'name' => t('Type'), 'default_sort' => 'asc');
+    $sort_options['created'] = array('method' => 'created', 'name' => t('Date created'), 'default_sort' => 'asc');
+    $sort_options['latest_activity'] = array('method' => 'latest_activity', 'name' => t('Latest build'), 'default_sort' => 'desc');
+  }
+
+  return $sort_options;
+}
+
+/**
+ * Handles the filters form submit.
+ */
+function installer_browser_filters_form_submit($form, &$form_state) {
+  $type = $form_state['values']['project_type'];
+  if ($form_state['clicked_button']['#id'] == 'edit-clear') {
+    $_SESSION['installer_browser_text_filter_' . $type] = array();
+  }
+  else {
+    $_SESSION['installer_browser_text_filter_' . $type] = $form_state['values']['search_text'];
+  }
+}
+
+/**
+ * Shows an install button for the Install Queue block.
+ */
+function installer_browser_advanced_block($form, &$form_state, $type) {
+  $advanced['advanced'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Advanced'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+  $advanced['advanced']['links'] = array(
+    '#theme' => 'links',
+    '#links' => array(
+      'manual_install' => array(
+        'title' => 'Manual install',
+        'href' => 'admin/modules/manual-install',
+        'attributes' => array(
+          'class' => array('use-ajax'),
+          'id' => array('manual-link'),
+          'data-dialog' => 'true',
+          'data-dialog-options' => json_encode(array('width' => 700, 'dialogClass' => 'project-dialog')),
+        ),
+      ),
+      'bulk_install' => array(
+        'title' => 'Install by names',
+        'href' => 'admin/modules/installer-browser/bulk/' . $type,
+        'attributes' => array(
+          'class' => array('use-ajax'),
+          'id' => array('bulk-link'),
+          'data-dialog' => 'true',
+          'data-dialog-options' => json_encode(array('width' => 700, 'dialogClass' => 'project-dialog')),
+        ),
+      ),
+    ),
+  );
+
+  return $advanced;
+}
+
+/**
+ * Install multiple projects by listing machine names.
+ */
+function installer_browser_bulk_install_form($form, &$form_state, $type) {
+  $form['type'] = array(
+    '#type' => 'hidden',
+    '#value' => $type,
+  );
+  $form['bulk'] = array(
+    '#type' => 'textarea',
+    '#required' => TRUE,
+    '#title' => t('Names'),
+    '#description' => t('Enter <em>%project</em> project names, one name per line. Names must only contain lowercase letters, numbers, and underscores.', array('%project' => $type)),
```

Comment:
No need for `<em>` tags here, they are added automatically when using a `%` placeholder.